coreneuron/mechanism/mechanism.hpp

```diff
@@ -16,6 +16,7 @@
 namespace coreneuron {
 // OpenACC with PGI compiler has issue when union is used and hence use struct
 // \todo check if newer PGI versions has resolved this issue
+// OL211214: bump
```

Comment:
Probably just drop this comment. We are already using the `union` in OpenMP implementation that we plan to take forward.